fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -833,6 +834,9 @@ public boolean getUseScanBlockCache() {
     @VarAttr(name = ENABLE_RULE_BASED_MATERIALIZED_VIEW_REWRITE)
     private boolean enableRuleBasedMaterializedViewRewrite = true;
 
+    @VarAttr(name = ANALYZE_FOR_MV)
+    private String analyzeForMV = "sample";
```

Comment:
```suggestion
    private String analyzeTypeForMV = "sample";
```

Reply:
maybe we should provide the users valid options? sample/full?

Reply:
done